src/store/getters/state-getters.ts

```diff
@@ -171,7 +171,7 @@ export const getCorrectionErrorType = (state: StateIF): CorrectionErrorTypes =>
 }
 
 /** The correction (business) start date (YYYY-MM-DD). */
-export const getCorrectionStartDate = (state: StateIF): string => {
+export const getCorrectedStartDate = (state: StateIF): string => {
```

Comment:
I think this is incorrect.

The corrected start date is `correctionInformation.correctedFilingDate`. This is the date of the original filing (the filing being corrected).

This getter is supposed to return the start date in this correction filing (the filing doing the correcting), if the user corrected it.

Maybe it would make more sense to name this "business start date"? (It's still called "start date" the schema though.)

Reply:
The founding date is the original business start date I think we should have the getter indicate that this is the corrected start date so someone doesn't accidentally use this getter thinking it is "business start date" maybe "corrected business start date"?

Reply:
Yes, the founding date is the original business start date. Agreed, this isn't that value. (And this is also not the corrected filing's date.) So we should not confuse things by using "corrected" in the name.

The naming convention for correction fields is "correctionXxx", so I rather like keeping this getter named getCorrectionStartDate, but I'm open to other ideas. What do you think?

Reply:
That makes sense to keep it consistent with the current naming conventions. I'll switch it back to getCorrectionStartDate.

Reply:
Done.